### PR TITLE
docs: add note about windows+powershell

### PR DESCRIPTION
**Pull request by RafaelGSS**
Address: https://github.com/clinicjs/node-clinic/issues/313﻿

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,16 @@ optimizations on background threads. With worker threads, the CPU will also
 utilize more than 100%. The visible percentage is always the combination of all
 these factors together.
 
+### Windows + PowerShell
+
+In order to diagnostic your application with node clinic, is mandatory execute your application after double hyphen(`--`),
```
Comment (simoneb) [inline README.md:72]:
```suggestion
In order to diagnose your application with node clinic, you should execute your application after double hyphens (`--`),
```

File: README.md [hunk 2]
```diff
@@ -67,6 +67,16 @@ optimizations on background threads. With worker threads, the CPU will also
...
+e.g: `clinic doctor -- node myapplication.js`.
+
+However, if you are using Windows and PowerShell need to know that a double hyphen instructs PowerShell to
+treat everything coming after as literal arguments rather than options.
+
+To avoid that behavior you can either quote ("--", '--') or escape (`--) the double hyphen
```
Comment (simoneb) [inline README.md:78]: we're missing a backtick here

File: README.md [hunk 3]
```diff
@@ -67,6 +67,16 @@ optimizations on background threads. With worker threads, the CPU will also
 utilize more than 100%. The visible percentage is always the combination of all
 these factors together.
 
+### Windows + PowerShell
+
+In order to diagnostic your application with node clinic, is mandatory execute your application after double hyphen(`--`),
+e.g: `clinic doctor -- node myapplication.js`.
+
+However, if you are using Windows and PowerShell need to know that a double hyphen instructs PowerShell to
+treat everything coming after as literal arguments rather than options.
```
Comment (simoneb) [inline README.md:76]: the english is not great here, rephrase

Comment (RafaelGSS) [reply README.md]: I have slightly changed, could you check?

**Review by simoneb — APPROVED** [review]

**Pull request merged**